src/utils/api.js

```diff
@@ -23,4 +23,5 @@ export const getArticles = () => {
         .catch(() => {
             console.log('err')
         })

```

Comment:
catch blocks shouldn't happen in this file because the end of the promise chain is actually in the component files that use these functions, I'd do error handling in the component files which opens you up to introduce error patterns